sdk/tables/Azure.Data.Tables/src/swagger/TableRestClient.cs

```diff
@@ -2,6 +2,7 @@
 // Licensed under the MIT License.
 
 using Azure.Core;
+using Azure.Data.Tables.Models;
```

Comment:
Just noticed you have files in the `swagger` directory. I think it might be better to move them into the project root. Historically the swagger directory was used to store swagger.json files when they had to be saved in the repo.

Reply:
My thought here was to leave them here as a reminder that these are generally work arounds that could be eliminated with fixes to the swagger or generator. Thoughts?

Reply:
That's a bit unusual, don't have any strong objections except for worry that people might not be able to find them.

I usually go with filing issues and referencing them from source but it's up to you here.